component-catalog-app/Examples/Tabs.elm

```diff
@@ -230,19 +246,40 @@ buildTooltip openTooltipId withTooltips id =
         ]
     , Tabs.build { id = id, idString = tabIdString }
         ([ Tabs.tabString tabName
-         , panelName
-            |> List.repeat 50
-            |> String.join "\n"
-            |> Html.text
-            |> List.singleton
-            |> Html.pre []
-            |> Tabs.panelHtml
+         , Tabs.panelHtml
+            (Html.pre
+                [ css
+                    [ Css.marginTop Css.zero
+                    , Css.padding (Css.px 10)
+                    , case config.fadeToPanelBackgroundColor of
+                        Nothing ->
+                            Css.batch []
+
+                        Just color ->
+                            Css.backgroundColor (colorToCss color)
+                    ]
+                ]
+                [ case config.fadeToPanelBackgroundColor of
+                    Nothing ->
+                        Html.text ""
+
+                    Just _ ->
+                        Message.view
```

Comment:
👍 Love this message! Will definitely help keep it clear what the scope of `fadeToPanelBackgroundColor` is